src/utils/constants.hpp

```diff
@@ -0,0 +1,228 @@
+//========================================================================================
+// (C) (or copyright) 2020. Triad National Security, LLC. All rights reserved.
+//
+// This program was produced under U.S. Government contract 89233218CNA000001 for Los
+// Alamos National Laboratory (LANL), which is operated by Triad National Security, LLC
+// for the U.S. Department of Energy/National Nuclear Security Administration. All rights
+// in the program are reserved by Triad National Security, LLC, and the U.S. Department
+// of Energy/National Nuclear Security Administration. The Government is granted for
+// itself and others acting on its behalf a nonexclusive, paid-up, irrevocable worldwide
+// license in this material to reproduce, prepare derivative works, distribute copies to
+// the public, perform publicly and display publicly, and to permit others to do so.
+//========================================================================================
+
+#ifndef UTILS_CONSTANTS_HPP_
+#define UTILS_CONSTANTS_HPP_
+
+#include <cmath>
+#include <string>
+
+namespace parthenon {
+namespace constants {
+
+// Conversion factors from the SI unit system
+struct SI {
```

Comment:
The SI units are order unity here because the physical constants are in SI in the `PhysicalConstants` class, correct? This doesn't constrain us to use SI units as order unity for the code units, right?

Reply:
Yes that's correct, SI is just the unit system used by the CODATA site where I copied and pasted the values here from. To keep your dynamical variables order unity you could use whatever unit system you want.